doc/GuideForCommitters.md

```diff
@@ -93,19 +93,19 @@ build, add a comment to the pull request that follows the syntax:
 Where `{platform-list}` is a comma-separated list of platforms chosen from the
 following tags:
 
-| Tag      | Platform |
-| :----    | :------- |
-| zlinux   | 64-bit Linux on Z |
-| zos      | 64-bit zOS |
-| plinux   | 64-bit Power Linux LE |
-| aix      | 64-bit AIX |
-| x32linux | 32-bit Linux |
-| xlinux   | 64-bit Linux |
-| xcrlinux | 64-bit Compressed References Linux |
-| win      | 64-bit Windows |
-| osx      | 64-bit macOS |
-| arm      | 32-bit AArch32 (cross-compile build only) |
-| aarch64  | 64-bit AArch64 (cross-compile build only) |
+| Alias    | Platform                                           | Spec Name           |
+| :----    | :-------                                           | :--------           |
+| arm      | 32-bit Linux on AArch32 (cross-compile build only) | linux_arm           |
+| x32linux | 32-bit Linux on x86                                | linux_x86           |
+| xlinux   | 64-bit Linux on x86-64                             | linux_x86-64        |
+| plinux   | 64-bit Linux on Power LE                           | linux_ppc-64_le_gcc |
+| zlinux   | 64-bit Linux on Z                                  | linux_390-64        |
+| aarch64  | 64-bit Linux on AArch64 (cross-compile build only) | linux_aarch64       |
+| riscv    | 64-bit Linux on RISC-V (cross-compile build only)  | linux_riscv64_cross |
+| zos      | 64-bit zOS                                         | zos_390-64          |
+| aix      | 64-bit AIX                                         | aix_ppc-64          |
+| win      | 64-bit Windows                                     | win_x86-64          |
+| osx      | 64-bit macOS                                       | osx_x86-64          |
```

Comment:
The original ordering of this table organized the targets by platform (I think).  I believe your ordering is by bitness first and then operating system which seems less useful to me.  I wonder if it would be better to organize by processor architecture because when you're selecting targets to test that is usually the main consideration.  Perhaps add dividing lines in the table between each category of architecture would help distinguish them.

Reply:
Organized by architecture. Markdown tables don't allow cells to be merged. So, I couldn't add dividing lines/rows in the table. Instead, I have added a separate column for architecture.